server/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/SnapshotBlocksIT.java

```diff
@@ -84,6 +100,90 @@ protected void setUpRepository() throws Exception {
         ensureSearchable();
     }
 
+    public void testSnapshotTotalAndDifferenceSizes() throws IOException {
```

Comment:
this test does not really fit into this class (it's about blocks). Can you move it to `DedicatedClusterSnapshotRestoreIT`?